test/src/tests-identity.js

```diff
@@ -1030,32 +1009,35 @@ describe('identity', function() {
 
         mParticle.init(apiKey, window.mParticle.config);
 
-        mockServer.requests = [];
-        mParticle.logEvent('Test1');
-        var event1 = getEvent(mockServer.requests, 'Test1');
-        Should(event1).not.be.ok();
+        window.fetchMock._calls = [];
+        mParticle.logEvent('Test Event1');
 
+        var testEvent1 = findEventFromRequest(window.fetchMock._calls, 'Test Event1');
+        
+        Should(testEvent1).not.be.ok();
+        
         mockServer.respondWith(urls.login, [
             200,
             {},
             JSON.stringify({ mpid: 'otherMPID', is_logged_in: false }),
         ]);
 
-        mParticle.logEvent('Test2');
+        mParticle.logEvent('Test Event2');
         mParticle.Identity.login();
-
         // server requests will have AST, sessionStart, Test1, Test2, and login
-        mockServer.requests.length.should.equal(5);
-        event1 = getEvent(mockServer.requests, 'Test1');
-        var event2 = getEvent(mockServer.requests, 'Test2');
-        var event3 = getEvent(mockServer.requests, 10);
-        var event4 = getEvent(mockServer.requests, 1);
-        var event5 = getIdentityEvent(mockServer.requests, 'login');
-        Should(event1).be.ok();
-        Should(event2).be.ok();
-        Should(event3).be.ok();
-        Should(event4).be.ok();
-        Should(event5).be.ok();
+        testEvent1 = findEventFromRequest(window.fetchMock._calls, 'Test Event1');
+        window.fetchMock._calls.length.should.equal(4);
+        
+        var testEvent2 = findEventFromRequest(window.fetchMock._calls, 'Test Event2');
+        var ASTEvent = findEventFromRequest(window.fetchMock._calls, 'application_state_transition');
+        var sessionStartEvent = findEventFromRequest(window.fetchMock._calls, 'session_start');
+        var loginEvent = getIdentityEvent(mockServer.requests, 'login');
+
+        Should(testEvent1).be.ok();
+        Should(testEvent2).be.ok();
+        Should(ASTEvent).be.ok();
+        Should(sessionStartEvent).be.ok();
+        Should(loginEvent).be.ok();
```

Comment:
Same here. Let's stop using `Should`

```suggestion
        expect(testEvent1).to.be.ok;
        expect(testEvent2).to.be.ok;
        expect(ASTEvent).to.be.ok;
        expect(sessionStartEvent).to.be.ok;
        expect(loginEvent).to.be.ok;
```

Reply:
Per above comment, out of scope